environments/kubernetes/manifests/thanos-rules-prometheusrule.yaml

```diff
@@ -138,9 +137,9 @@ spec:
           $value }} seconds for HTTP requests.
       expr: |
         (
-          histogram_quantile(0.99, sum by (job, le) (http_request_duration_seconds_bucket{job=~"thanos-receive.*", namespace="telemeter-stage", handler="receive"})) > 10
+          histogram_quantile(0.99, sum by (job, le) (http_request_duration_seconds_bucket{job=~"thanos-receive-.*", handler="receive"})) > 10
         and
-          sum by (job) (rate(http_request_duration_seconds_count{job=~"thanos-receive.*", namespace="telemeter-stage", handler="receive"}[5m])) > 0
+          sum by (job) (rate(http_request_duration_seconds_count{job=~"thanos-receive-.*", handler="receive"}[5m])) > 0
```

Comment:
Note that this regex match will also select the controllers. Not sure if that's intended.

Reply:
Nvm I totally missed the handler selector.

Reply:
That would be true if we didn't have the handler="receive". So we should be fine.